ctr-dev-seed-data.sql

```diff
@@ -0,0 +1,80 @@
+# ************************************************************
```

Comment:
I think these two sql files should probably live somewhere within `/api` since that's what cares about them. `/api/db` perhaps?